src/main/java/com/auth0/json/mgmt/ResourceServer.java

```diff
@@ -0,0 +1,183 @@
+package com.auth0.json.mgmt;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ResourceServer {
+    @JsonProperty
+    private String id;
+    @JsonProperty
+    private String name;
+    @JsonProperty("identifier")
+    private String identifier;
+    @JsonProperty("scopes")
+    private List<Scope> scopes;
+    @JsonProperty("signing_alg")
+    private String signingAlg;
+    @JsonProperty("signing_secret")
+    private String signingSecret;
+    @JsonProperty("allow_offline_access")
+    private Boolean allowOfflineAccess;
+    @JsonProperty("skip_consent_for_verifiable_first_party_clients")
+    private Boolean skipConsentForVerifiableFirstPartyClients;
+    @JsonProperty("token_lifetime")
+    private Integer tokenLifetime;
+    @JsonProperty("verificationKey")
+    private String verificationKey;
+    @JsonProperty("verificationLocation")
+    private String verificationLocation;
+    @JsonProperty("is_system")
+    private Boolean isSystem;
+
+    public ResourceServer() {
```

Comment:
I don't remember if this constructor must exist because of Jackson, see if it can be removed. Also we need a constructor that receives at least the `identifier` parameter as that's the only required parameter when **creating a new** ResourceServer instance to post. `id` will always be **returned by the server**, but I trust Jackson is going to handle it just fine 😄 . (Add those tests)

Reply:
Default constructor removed.
As I understand JSON transformation is covered in `ResourceServerEntityTest` when content of `resource_servers_list.json` is converted to `ResourceServer` instances. Do we nee additional tests for JSON conversion?